apps/fp-frontend-app/src/fagsakprofile/fagsakProfileIndex.module.css

```diff
@@ -1,3 +1,18 @@
 .panelPadding {
   padding: 20px;
 }
+
+/* hidden */
+.historikkPanelUtvid {
```

Comment:
Desse er vel ikkje spesifikke for historikkpanelet? Burde kanskje heller vore kalla skjulFagsakInfo og visFagsakInfo (eller noko sånt)